tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/ConfidentialClientIntegrationTests.cs

```diff
@@ -628,7 +634,66 @@ private async Task RunOnBehalfOfTestAsync(LabResponse labResponse)
                 .WithAuthority(new Uri(oboHost + authResult.TenantId), true)
                 .WithClientSecret(secret)
                 .WithTestLogging()
-                .Build();
+                .BuildConcrete();
+
+            var userCacheRecorder = confidentialApp.UserTokenCache.RecordAccess();
+
+            UserAssertion userAssertion = new UserAssertion(authResult.AccessToken);
+
+            string atHash = userAssertion.AssertionHash;
+
+            authResult = await confidentialApp.AcquireTokenOnBehalfOf(s_scopes, userAssertion)
+                .ExecuteAsync(CancellationToken.None)
+                .ConfigureAwait(false);
+
+            MsalAssert.AssertAuthResult(authResult, user);
+            Assert.AreEqual(atHash, userCacheRecorder.LastAfterAccessNotificationArgs.SuggestedCacheKey);
+
+#pragma warning disable CS0618 // Type or member is obsolete
+            await confidentialApp.GetAccountsAsync().ConfigureAwait(false);
```

Comment:
>GetAccountsAsync [](http://example.com/codeflow?start=34&length=16)

this can't be used w/cca

Reply:
This was a check used by a different OBO test in same the file. is was accidently copied over. not need. I'm am not sure why it was placed in the other test however.